en/components/date_time_editor.md

```diff
@@ -84,7 +84,7 @@ IgxDateTimeEditor supports different display and input formats.
 
 For display, it uses Angular's [DatePipe](https://angular.io/api/common/DatePipe) which allows it to support predefined format options, such as `shortDate` and `longDate`. It can also accept a constructed format string using characters supported by the DatePipe, e.g. `EE/MM/yyyy`. Notice that formats like shortDate, longDate, etc., can be used as `displayFormat` only.
```

Comment:
```suggestion
It uses Angular's [DatePipe](https://angular.io/api/common/DatePipe), which allows it to support predefined format options, such as `shortDate` and `longDate`. It can also accept a constructed format string using characters supported by the DatePipe, e.g. `EE/MM/yyyy`. Notice that formats like shortDate, longDate, etc., can be used as `displayFormat` only.
```